mlir/lib/Dialect/SparseTensor/Transforms/Sparsification.cpp

```diff
@@ -1032,10 +1031,14 @@ static bool getAllTidLvlsInLatPoints(
       });
 
   if (isDenseLT(env.lt(outTid, curr))) {
+    auto stt = getSparseTensorType(env.op().getOutputs().front());
     // Note that we generate dense indices of the output tensor
```

Comment:
can you horizontally align the comment a bit (avoid running long, then short in col space)